TCMSApp/src/client/app/tests/tests.html

```diff
@@ -1,88 +1,83 @@
+                    <li class="list-group-item">
+                        <button type="button" class="btn btn-default">
+                           Create suite
+                        </button>
+                    </li>
+                    <li class="list-group-item">Unsuited tests
+                        <button type="button" class="btn btn-default">
+                            <span class="glyphicon glyphicon-plus"></span>
+                        </button>
+                    </li>
+                    <li class="list-group-item">{{ vm.f.suiteName }}
+                        <button type="button" class="btn btn-default">
+                            <span class="glyphicon glyphicon-plus"></span>
+                        </button>
+                    </li>
+                </ul>
+            </div>
 
-                        <div class="row">
-                            <div class="col-md-offset-1 col-md-2">Description</div>
-                            <div class="col-md-8 well text-justify">
-                                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas et ultrices magna. Nam sit amet est id felis scelerisque luctus.
-                                Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Nam porttitor leo id nisi fermentum, eu fermentum nibh mattis.
+            <div class="col-md-9">
+                <div class="well" ng-repeat="tc in vm.f.tests">
+                    <div class="row">
+                        <div class="col-md-5">
+                            <div class="checkbox">
+                                <h3>
+                                <label>
+                                    <input type="checkbox">{{ tc.testName }}
+                                </label>
+                                </h3>
                             </div>
                         </div>
+                        <div class="col-md-7"></div>
+                    </div>
 
-                        <div class="row">
-                            <div class="col-md-offset-1 col-md-2"><strong>Priority:</strong> High</div>
-                            <div class="col-md-offset-1 col-md-2"><strong>Date:</strong> 23.10.15</div>
-                            <div class="col-md-offset-1 col-md-2"><strong>Bugs: </strong><div class="badge"> 15</div></div>
-                            <div class="col-md-offset-1 col-md-2"><strong>Steps: </strong><div class="badge"> 15</div></div>
+                    <div class="row">
+                        <div class="col-md-offset-1 col-md-2">Description</div>
+                        <div class="col-md-8 well text-justify">
+                            {{ tc.testDescription }}
                         </div>
                     </div>
-                </div>
 
+                    <div class="row">
+                        <div class="col-md-offset-1 col-md-2"><strong>Priority:</strong> High</div>
```

Comment:
why all this data is not from our fakers?

Priority: High
Date: 23.10.15
Bugs: 15
Steps: 15